WORKSPACE

```diff
@@ -323,12 +323,21 @@ go_rules_compat(
 # clang + llvm 8.0
 http_archive(
     name = "clang",
-    build_file = "//toolchain:clang.BUILD",
+    build_file = "//toolchain:all_files.BUILD",
     sha256 = "0f5c314f375ebd5c35b8c1d5e5b161d9efaeff0523bac287f8b4e5b751272f51",
     strip_prefix = "clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04",
     url = "http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz",
 )
 
+# sysroot for arm
```

Comment:
Could you expand this comment?  Is this a host=linux target=arm compiler?

Reply:
Done